sokol_gfx.h

```diff
@@ -146,14 +146,63 @@
         beginning a pass will reset the viewport to the size of the framebuffer used
         in the new pass,
 
-    --- to update the content of buffer and image resources, call:
+    --- to overwrite the content of buffer and image resources, call:
```

Comment:
To keep the consistency between the API and the documentation, would you to rollback to the word `update` and put in parenthesis the world `overwrite`? 

```
--- to update (overwrite) the content of buffer and image resources, call:
```

Reply:
yes good idea